Cargo.toml

```diff
@@ -111,8 +102,13 @@ magicblock-accounts-api = { path = "./magicblock-accounts-api" }
 magicblock-accounts-db = { path = "./magicblock-accounts-db" }
 magicblock-api = { path = "./magicblock-api" }
 magicblock-bank = { path = "./magicblock-bank" }
+magicblock-committor-program = { path = "./magicblock-committor-program", features = [
```

Comment:
nitpick: we now have services ending with `committor` and `committer`, like `RemoteAccountCommitter`. `committor` must be a typo